docs/demo/gaiden-css/components/page-banner/demo.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/gaiden-css/ninja-gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+  <style>
+    body {
+      background-color: #eff4fa;
+    }
+  </style>
+</head>
+
+<body>
+  <section class="page-banner" role="banner">
+    <picture class="picture picture--cover">
+      <source srcset="/demo/gaiden-css/images/background-header-desktop.jpg" media="(min-width: 1200px)">
+      <source srcset="/demo/gaiden-css/images/background-header-large.jpg" media="(min-width: 960px)">
+      <source srcset="/demo/gaiden-css/images/background-header-medium.jpg" media="(min-width: 768px)">
+      <source srcset="/demo/gaiden-css/images/background-header.jpg" media="(max-width: 767px)">
+      <img src="/demo/gaiden-cssimages/background-header-medium.jpg" class="picture__image" alt="Imagem " role="img">
```

Comment:
@bsoares Maybe yes... but on the https://www.w3.org/TR/wai-aria/roles#img they are describing `role="img"` as a "container for a collection of elements that form an image". In this case, we haven't a label to describe this image. So, you're right that, here, it is not necessary this role :)